mlrun/db/httpdb.py

```diff
@@ -3788,6 +3789,7 @@ def list_model_endpoints(
         :param top_level:       Whether to return only top level model endpoints.
         :param uids:            A list of unique ids to filter by.
         :param latest_only:     Whether to return only the latest model endpoint version.
+        :param metrics:         A list of metrics to add. Defaults to all metrics.
```

Comment:
If we keep two different params, please add here that this is relevant only when `tsdb_metrics=True`